src/composables/useSearch.ts

```diff
@@ -81,7 +81,7 @@ export const useSearch = () => {
       return initialSearchItems.value.slice(0, 100);
     }
 
-    const word = extractOnlySearchWord.value || "h"; // "h" is included in all URLs;
+    const word = extractOnlySearchWord.value || "http"; // "http" is included in all URLs;
```

Comment:
neccessary more than 2 charactor because `minMatchCharLength` is set to 2.

Reply:
Very nice, Thanks!